joplin/base/migrations/0003_auto_20171128_1944.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.7 on 2017-11-28 19:44
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import modelcluster.contrib.taggit
+import modelcluster.fields
+import wagtail.wagtailcore.blocks
+import wagtail.wagtailcore.fields
+import wagtail.wagtailsnippets.blocks
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailcore', '0040_page_draft_title'),
+        ('taggit', '0002_auto_20150616_2121'),
+        ('base', '0002_create_homepage'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ApplicationBlock',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('url', models.URLField()),
+                ('description', models.TextField()),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='Contact',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255)),
+                ('email', models.EmailField(max_length=254)),
+                ('phone_number', models.CharField(max_length=255)),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='Department',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255)),
+                ('address', models.CharField(max_length=255)),
+                ('hours', models.CharField(max_length=255)),
+                ('phone_number', models.CharField(max_length=255)),
+                ('email', models.EmailField(max_length=254)),
+                ('description', models.TextField()),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='Director',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255)),
+                ('photo', models.CharField(max_length=255)),
+                ('email', models.EmailField(max_length=254)),
```

Comment:
don't think this matters at all but why is this magic number different from the rest?

Reply:
You have to manually specify`CharField.max_length` and 255 is a pretty common default. Django chooses a sensible default for `EmailField.max_length` which is apparently 254 :|

Reply:
I think a decent interim solution here would be to just set pseudo constants in the class:
```
class Migration(migrations.Migration):

    DEFAULT_CHAR_FIELD_MAX = 255
    DEFAULT_EMAIL_FIELD_MAX = 254
```

Reply:
These migrations are auto-generated from models, so I'll probably punt on doing that here. I'll add some constants to `models.py` so magic numbers aren't scattered everywhere.